benchmarking/frameworks/framework_base.py

```diff
@@ -152,32 +153,35 @@ def runBenchmark(self, info, benchmark, platform):
             run_result, _ = processRun([postprocess_cmd], shell=True)
             if run_result:
                 getLogger().info("Postprocessing output: %s", run_result)
-
         return output, output_files
 
     def composeProcessCommand(self, process_info, model, test, model_files):
         files_db = {"process": {"files": {}}}
         for f_key in process_info["files"]:
             f_value = process_info["files"][f_key]
             files_db["process"]["files"][f_key] = f_value["location"]
-
         return self._getReplacedCommand(process_info["command"],
                                    files_db["process"]["files"], model, test, model_files)
 
     @abc.abstractmethod
-    def composeRunCommand(self, platform, program, model, test, model_files,
+    def composeRunCommand(self, platform, programs, model, test, model_files,
                           input_files, output_files, shared_libs, preprocess_files=None):
-        if "arguments" not in test:
+        if "arguments" not in test and "command" not in test:
             return None
-
         files = input_files.copy() if input_files is not None else {}
         files.update(output_files if output_files is not None else {})
         files.update(preprocess_files if preprocess_files is not None else {})
-
-        command = test["arguments"]
-        command = self._getReplacedCommand(command, files, model, test,
+        files.update(programs if programs is not None else {})
+        if "arguments" in test:
+            command = test["arguments"]
+            command = self._getReplacedCommand(command, files, model, test,
+                                            model_files)
+            return '"' + program + '" ' + command
```

Comment:
we don't have program any more. 

I'm actually thinking that since "arguments" is deprecated, we can rewrite it very early, say in "benchmark.py" so that here we only need to deal with one case.